doc/weights.xml

```diff
@@ -0,0 +1,70 @@
+#############################################################################
+##
+#W  weights.xml
+#Y  Copyright (C) 2014-19                               Raiyan Chowdhury
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+
+<#GAPDoc Label="EdgeWeights">
+    <ManSection>
+    <Attr Name="EdgeWeights" Arg="digraph"/>
+    <Oper Name="EdgeWeightsMutableCopy" Arg="digraph"/>
+    <Returns>A list of lists of integers, floats or rationals.</Returns>
+    <Description>
+     <C>EdgeWeights</C> returns the list of lists of edge weights of
+    each edge of the digraph <A>digraph</A>.
+
+    More specifically, a value <C>j</C> appears in <C>weights[i]</C> each time
+    there exists the edge <C>[i, j]</C> in <A>digraph</A>. <P/>
+
+    The function <C>EdgeWeights</C> returns an immutable list of
+    lists, whereas the function <C>EdgeWeightsMutableCopy</C> returns a copy
+    of <C>EdgeWeights</C> which is a mutable list of mutable lists. <P/>
```

Comment:
Add a note explaining that the edge weights of a digraph cannot be computed: they are set using `SetEdgeWeights` or by calling `EdgeWeightedDigraph`.